src/chapters/strings/exercises.rst

```diff
@@ -0,0 +1,75 @@
+String Exercises
+----------------
+
+1. Identify what is returned for each of the following:
+
+   a. 'JavaScript'[8]
+   b. “Strings are sequences of characters.”[5]
+   c. “Wonderful”.length
+   d. "Do spaces count?".length
+
+|
+
+2. Consider the following variables:
```

Comment:
How about replacing this with an exercise that uses template literals?

Reply:
Done.